orbit/pkg/table/orbit_info/orbit_info.go

```diff
@@ -77,5 +81,6 @@ func (o Extension) GenerateFunc(_ context.Context, _ table.QueryContext) ([]map[
 		"orbit_channel":       o.orbitChannel,
 		"osqueryd_channel":    o.osquerydChannel,
 		"desktop_channel":     o.desktopChannel,
+		"uptime":              time.Since(o.startTime).Round(1 * time.Second).String(),
```

Comment:
Orbit changes should be documented in orbit/changes.

Also, `orbit_info.yml` needs to be updated.